plot_pixel_values.py

```diff
@@ -95,7 +143,20 @@ def get_opt():
                     ax.autoscale_view()
                 else:
                     ax.plot(x, y)
-
+                ax.annotate("{}".format(y.name),
```

Comment:
This wasn't a bad idea, but it looks like it doesn't get called again if a new pixel is selected (should it be more obvious when a new pixel is selected anyway?).